src/ITensorNetworks/peps.jl

```diff
@@ -0,0 +1,142 @@
+"""
+A finite size PEPS type.
+"""
+mutable struct PEPS
+  data::Matrix{ITensor}
+end
+
+PEPS(Nx::Int, Ny::Int) = PEPS(Matrix{ITensor}(undef, Nx, Ny))
+
+"""
+    PEPS([::Type{ElT} = Float64, sites; linkdims=1)
+Construct an PEPS filled with Empty ITensors of type `ElT` from a collection of indices.
+Optionally specify the link dimension with the keyword argument `linkdims`, which by default is 1.
+"""
+function PEPS(::Type{T}, sites::Matrix{<:Index}; linkdims::Integer=1) where {T<:Number}
```

Comment:
You could simplify this function by making use of the `inds_network` function, like is shown here: https://github.com/ITensor/ITensorNetworkAD.jl/blob/main/examples/ITensorNetworks/peps.jl#L7-L25

Again, we could add this as a future issue, since it may requiring adapting that function a bit to make it work here.

Reply:
Made an issue here: #11